src/components/index/quote.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const quoteText = [
+    {
+        quote: "One of the best rolling release distros out there.",
+        author: "Sourav Rudra",
+        link: "https://news.itsfoss.com/rhino-linux-2024-2-release/",
+        source: "It's FOSS" 
+    },
+    {
+        quote: "Rhino Linux's latest release makes it the Swiss Army knife of Linux distributions.",
+        author: "Jack Wallen",
+        link: "https://www.zdnet.com/article/rhino-linux-latest-release-makes-it-the-swiss-army-knife-of-linux-distributions/",
+        source: "ZDNET"
+    },
+];
+
+function Quote() {
+    return (
+        <div className="md:w-[85%] m-auto grid grid-cols-1 gap-4 pb-8 md:pt-4">
```

Comment:
Please change to use grid-cols-2 on medium+large devices pls.

Reply:
Sure